### DISCUSSION TITLE
Can I use vuepress-plugin-md-enhance with standard vuepress2 theme?

### DISCUSSION THREAD

**Discussion by vmihailenco** [vuepress-theme-hope/vuepress-theme-hope#1619] [Q&A] [answered]
I've installed the plugin

```shell
yarn add vuepress-plugin-md-enhance@next -D
```

And updated config.ts:

```ts
    [
      'vuepress-plugin-md-enhance',
      {
        mermaid: true,
      },
    ],
```

Then I see the following in console:

```
runtime-core.esm-bundler.js?5c40:38 [Vue warn]: Component "CodeGroup" has already been registered in target app.
warn @ runtime-core.esm-bundler.js?5c40:38
runtime-core.esm-bundler.js?5c40:38 [Vue warn]: Component "CodeGroupItem" has already been registered in target app.
warn @ runtime-core.esm-bundler.js?5c40:38
clientAppEnhances.js:3 Uncaught (in promise) ReferenceError: Cannot access 'clientAppEnhances' before initialization
```

I am able to use mermaid, but `CodeGroup` does NOT work any more. Is it possible to use vuepress-plugin-md-enhance@next with standard vuepress2 theme?

PS https://vuepress-theme-hope.github.io/md-enhance returns 404.

**Comment by Mister-Hope** [comment, marked as answer]
Hi, where did you get that link?

The correct link should be https://vuepress-theme-hope.github.IO/v2/md-enhance/

**Reply by vmihailenco** [reply]
https://vuepress-theme-hope.github.io/guide/markdown/ the first paragraph

**Reply by Mister-Hope** [reply]
Try removing any service worker under `vuepress-theme-hope.github.io` installed in your browser. That's already a dead link.

**Reply by Mister-Hope** [reply]
The issue you reported is fixed in main, will be released soon

**Reply by vmihailenco** [reply]
Thanks :+1:

**Reply by Mister-Hope** [reply]
Hi, I am not sure whether you have removed all the invalid service workers. And if you haven't yet, I am looking for your feedback!!! 🙂


Please visit https://vuepress-theme-hope.github.io/guide/markdown/ again, and try wait for a little bit. If I am doing the correct thing, you should see a popup in the bottom right corner noticing you "New content available", and after you click the refresh buton, you should found that link be 404.

Since I no longer have any old service worker installed on my machine, I cannot test this.

**Reply by vmihailenco** [reply]
>and after you click the refresh buton, you should found that link be 404.

Yes, that works exactly like you've explained. I was able to find the docs I was looking for so no issues here.
